src/Microsoft.SqlTools.ServiceLayer/LanguageServices/ConnectedBindingQueue.cs

```diff
@@ -125,7 +125,14 @@ public void OpenConnections(string serverName, string databaseName, int millisec
             {
                 if (bindingContext.BindingLock.WaitOne(millisecondsTimeout))
                 {
-                    bindingContext.ServerConnection.Connect();
+                    try
+                    {
+                        bindingContext.ServerConnection.Connect();
```

Comment:
is it really safe to ignore failures trying to reconnect?  i.e. will this cause random failures in other feature areas?